go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	google.golang.org/grpc v1.63.3
 	google.golang.org/protobuf v1.34.2
 	gopkg.in/ini.v1 v1.67.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Pls add these changes in the respective commit itself.

Reply:
done